systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeKafkaExternalListenersST.java

```diff
@@ -213,6 +213,9 @@ private void testWeirdUsername(ExtensionContext extensionContext, String weirdUs
             .withTopicName(topicName)
             .withMessageCount(MESSAGE_COUNT)
             .withUserName(weirdUserName)
+            // we disable ssl.endpoint.identification.algorithm for external listener (i.e., NodePort),
+            // because TLS hostname verification is not supported on such listener type.
+            .withAdditionalConfig("ssl.endpoint.identification.algorithm=\n")
```

Comment:
Why is this using node ports?

Reply:
I think that we can re-write it to **not** use NodePort but such change I can be in another PR. The reason why it is using NodePort is answered by Lukas.

> ![image](https://user-images.githubusercontent.com/30839163/201052317-8f08edd3-c841-4437-9f2b-2e672f9b769b.png)

I can definitely, open another PR, which will change it :)

Reply:
I have created issue to have it tracked -> https://github.com/strimzi/strimzi-kafka-operator/issues/7608